src/app/pages/debrief/cat-a-mod1/debrief.cat-a-mod1.page.ts

```diff
@@ -28,7 +28,7 @@ export class DebriefCatAMod1Page implements OnInit {
   }
 
   async navigatePostDebriefHolding(): Promise<void> {
-    await this.routeByCat.navigateToPage(TestFlowPageNames.POST_DEBRIEF_HOLDING_PAGE, TestCategory.EUAM1);
+    await this.routeByCat.navigateToPage(TestFlowPageNames.POST_DEBRIEF_HOLDING_PAGE);
```

Comment:
is routeByCategoryProvider the right one to be using here? or should it be router.navigate([POST_DEBRIEF_HOLDING_PAGE])?

Reply:
@tomcrawleyy as the navigateToPage function simply calls router.navigate if u dont call it with a category, your right we could just call router.navigate directly.  I left it as this function so all navigation was consistant